internal/bminventory/inventory.go

```diff
@@ -1670,6 +1672,13 @@ func (b *bareMetalInventory) UpdateClusterInstallConfigInternal(ctx context.Cont
 			return common.NewApiError(http.StatusBadRequest, err)
 		}
 
+		if mirrorregistries.IsMirrorConfigurationSet(mirrorRegistryConfiguration) {
```

Comment:
Why do this check?

Reply:
Good catch! We also need to set it if it’s nil (in case the mirror configuration was removed)